scripts/pcmping

```diff
@@ -0,0 +1,167 @@
+#!/usr/bin/env python
+
+"""
+    Portchannel member ping: get portchannel member packet loss state.
+
+"""
+
+from scapy.all import *
+import argparse
+import random
+import socket
+import select
+import time
+import ipaddr as ipaddress
+import sys
+import swsssdk
+
+RCV_SIZE_DEFAULT = 4096
+RCV_TIMEOUT = 10000
+DEFAULT_PACKET_LEN = 100
+SELECT_TIMEOUT = 2
+
+def receive(my_socket, timeout, exp_pkt, exp_socket, time_sent):
+    timeLeft = timeout
+    delay = 0
+    while True:
+        startedSelect = time.time()
+        whatReady = select.select(my_socket, [], [], timeLeft)
+        if whatReady[0] == []: # Timeout
+            return False, delay
+        timeReceived = time.time()
+        
+        for sel in whatReady[0]:
+            recPacket = sel.recv(4096)
+            e = str(exp_pkt)
+            # exclude ethernet header
+            p = str(recPacket[14:])
+            if e == p:
+                if sel == exp_socket:
+                  delay = (timeReceived - time_sent) * 1000
+                  return True, delay
+        timeLeft = timeLeft - (timeReceived - startedSelect)
+        if timeLeft <= 0:
+            return False, delay
+        
+def create_decap_packet(interface, dst_out, dst_in, sockets, exp_socket, max_iter):
+    dscp = 16
+    tos = dscp << 2
+    tcp_hdr = TCP(sport=1234, dport=80, flags="S", chksum=0)
+    for x in range(1, max_iter):
+        print "---------- Attempting to create decap packet that goes through %s, iteration: %d ---------" % (interface, x)
+        ip_src = socket.inet_ntoa(struct.pack('>I', random.randint(1, 0xffffffff)))
+        ip_src =ipaddress.IPv4Address(unicode(ip_src,'utf-8'))
+        while ip_src == ipaddress.IPv4Address(unicode(dst_in,'utf-8')) or \
+              ip_src.is_multicast or ip_src.is_private or ip_src.is_reserved:
+            ip_src = socket.inet_ntoa(struct.pack('>I', random.randint(1, 0xffffffff)))
+            ip_src =ipaddress.IPv4Address(unicode(ip_src,'utf-8'))
+        inner_pkt  = IP(src=ip_src, dst=dst_in, tos=tos, ttl=64, id=1, ihl=None,proto=4)/tcp_hdr
+        inner_pkt = inner_pkt/("".join([chr(x) for x in xrange(DEFAULT_PACKET_LEN - len(inner_pkt))]))
+        pkt = IP(src='4.4.4.4', dst=dst_out, tos=tos)/inner_pkt
```

Comment:
>4.4.4.4 [](start = 22, length = 7)

why hardcode this one? can you use its own po ip? there is chance that dst_out is also 4.4.4.4